qemu/tests/virtio_scsi_mq.py

```diff
@@ -53,7 +56,9 @@ def proc_interrupts_results(results):
                         irq_des += " %s" % irq_item
                     count += 1
                 results_dict[irq_key]["irq_des"] = irq_des.strip()
-        return results_dict, cpu_list
+        if not irqs_key:
+            test.fail("Can't find virtio request interrupts from procfs")
```

Comment:
please use test.error() here, as it's a env problem.

Reply:
ok, I will update it.